README.md

```diff
@@ -15,16 +15,12 @@ buildscript {
   }
   dependencies {
     classpath 'com.vanniktech:gradle-maven-publish-plugin:0.14.2'
-    // For Kotlin projects, you need to add Dokka.
-    classpath 'org.jetbrains.dokka:dokka-gradle-plugin:1.x.x'
   }
 }
 
 apply plugin: "com.vanniktech.maven.publish"
 ```
 
-Information: [This plugin is also available on Gradle plugins](https://plugins.gradle.org/plugin/com.vanniktech.maven.publish)
```

Comment:
Removed this because our automation does not publish to this.

Reply:
Right. I used to publish this by hand